docker-cloud/cloud-swarm/index.md

```diff
@@ -78,7 +78,11 @@ To register an existing swarm in Docker Cloud:
 
     To register a swarm with an organization, prefix the new name with the organization name, for example `myorganization/myteamswarm`.
 
-The manager node pulls the `dockercloud/registration` container and runs it on the swarm. The swarm then appears in the **Swarms** screen in Docker Cloud.
+The manager node pulls the `dockercloud/registration` container which creates a
+global service called `dockercloud/serverproxy`. This service runs on _all_ of
```

Comment:
the service name is `dockercloud-server-proxy`. The image it uses is `dockercloud/server-proxy``